python/lsst/ts/adamSensors/model.py

```diff
@@ -26,13 +26,13 @@ def __init__(self, log, simulation_mode=False):
         self.log = log
```

Comment:
Consider allowing for log to be `None` and also to create a child logger:

in line 22:

```
def __init__(self, log=None, simulation_mode=False):
```

Then here you do:

```
  if log is None:
      self.log = logging.getLogger(type(self).__name__)
  else:
      self.log = log.getChild(type(self).__name__)
```

you will need to `import logging` as well.